azure-core/azure/core/pipeline/policies/universal.py

```diff
@@ -280,7 +280,12 @@ def deserialize_from_http_generics(cls, response):
         # Try to use content-type from headers if available
         content_type = None
         if 'content-type' in response.headers:
-            content_type = response.headers['content-type'].split(";")[0].strip().lower()
+            try:
+                content_type_header = response.headers['content-type'].split(";")[0]
```

Comment:
Is it a problem that we drop the charset? Also, we can probably specify maxsplit=1 if we are to drop everything anyways...

Reply:
This logic has worked in out current SDKs - adding @lmazuel is case he has any thoughts?